src/sql/src/plan/notice.rs

```diff
@@ -37,6 +37,7 @@ pub enum PlanNotice {
         key: Vec<ColumnName>,
         name: String,
     },
+    WebhookValidationDoesNotUseColumns,
```

Comment:
This should just be an error because it's never correct.

Reply:
Done!